data/src/main/scala/com/exini/dicom/data/Elements.scala

```diff
@@ -91,8 +90,8 @@ case class Elements(characterSets: CharacterSets, zoneOffset: ZoneOffset, data:
   def getValueElement(tagPath: TagPathTag): Option[ValueElement] = getPath(tagPath, Option.apply)
   def getValue(tag: Int): Option[Value]                          = getValueElement(tag).map(_.value)
   def getValue(tagPath: TagPathTag): Option[Value]               = getValueElement(tagPath).map(_.value)
-  def getBytes(tag: Int): Option[ByteString]                     = getValue(tag).map(_.bytes)
-  def getBytes(tagPath: TagPathTag): Option[ByteString]          = getValue(tagPath).map(_.bytes)
+  def getBytes(tag: Int): Option[Array[Byte]]                    = getValue(tag).map(_.bytes)
+  def getBytes(tagPath: TagPathTag): Option[Array[Byte]]         = getValue(tagPath).map(_.bytes)
```

Comment:
Maybe we could use `Seq[Byte]` for public API? (parser, elements...) We would prevent users from mutating the bytes and afaik Array should be implicitly converted to an immutable structure.

Reply:
Good question. See separate PR. One note is that I went with immutable.ArraySeq instead of boxing as I have seen some evidence that indirection is vastly more performant compared to constant boxing/unboxing.